.pipelines/templates/.builder-release-template-windows.yaml

```diff
@@ -37,6 +37,10 @@ parameters:
   - name: windowsCoreImageUrl
     displayName: Windows core base container image URL Override
     type: string
+  - name: windowsContainerImageJsonUrl
+    displayName: Windows container image JSON URL Override
+    type: string
+    default: https://wcctagentbakerstorage.blob.core.windows.net/simship/2025/06B/payload.json
```

Comment:
The default should be an empty string or else it's possible that all pipelines will use this json.